src/ZipRangeReader.php

```diff
@@ -250,4 +252,54 @@ private function correctCRC(string $crc): string
     {
         return strtoupper(implode(array_reverse(str_split($crc, 2))));
     }
+
+    /**
+     * Return a file pointer to a decompressed byte stream of the file at `$path`.
+     *
+     * @param string $path The file path within the zip.
+     * @return resource A file pointer to the decompressed byte stream.
+     * @throws FileNotFound Thrown if the requested file path does not exist in the zip.
+     * @throws UnsupportedCompression Thrown if the file is compressed with something other than DEFLATE.
+     */
+    public function getStream(string $path) {
+        // Try and retrieve the file from the directory information
+        $file = @$this->files[$path];
+        if(!$file) {
+            throw new FileNotFound("File " . $path . " not found in the zip file.");
+        }
+
+        // Retrieve the Local File Header
+        $header = $this->seeker->retrieveStart(30, $file["offset"]);
+        $file["local_header"] = $this->readLocalHeader($header);
+
+        // Calculate the offset of the compressed data (File offset + LFH length + file name length + extra field length)
+        $data_offset = $file["offset"] + 30 + $file["local_header"]["fnlength"] + $file["local_header"]["eflength"];
```

Comment:
cool

Reply:
Thinking more, we might be able to dodge having to request and retrieve the local file header, since we only read it here to get the length of the filename and the extra field, and the type of compression used, all of which we already have from the Central Directory.

So if we can trust that the Central Directory and the Local File Header will always match, there might be a bit more efficiency to be squeezed out here. Probably doesn't make much difference for local files, but it would save one read for remote files.

Reply:
I will add a separate ticket for this, since it only affects calls made by the main library, and thus can be added independently of Seeker implementations without screwing anything up or requiring modifications to the Seekers.

Reply:
Added #12